hub/core/compute/process.py

```diff
@@ -6,6 +6,11 @@ class ProcessProvider(ComputeProvider):
     def __init__(self, workers):
         self.workers = workers
         self.pool = ProcessPool(nodes=workers)
```

Comment:
Previously noticed issues with ray pool including reporting of errors. Wanted to make sure if you have observed any significant issue during testing?

going with default ray compute workers from my experience have been more stable (at least more transparent on the exceptions)